gulpfile.js

```diff
@@ -325,6 +335,20 @@ function release_win32() {
     return archive.finalize();
 }
 
+//Create distribution package for windows 64bit platform
+function release_win64() {
+    var src = path.join(appsDir, pkg.name, 'win64');
+    var output = fs.createWriteStream(path.join(releaseDir, get_release_filename('win64', 'zip')));
+    var archive = archiver('zip', {
+        zlib: { level: 9 }
+    });
+    archive.on('warning', function (err) { throw err; });
+    archive.on('error', function (err) { throw err; });
+    archive.pipe(output);
+    archive.directory(src, 'Betaflight Configurator');
+    return archive.finalize();
+}
+
```

Comment:
I don't know if it's better to integrate this into the `release_win32()` passing a parameter. I have left it in another function to maintain the style.

Reply:
Yes, this is the way to go. Adding parameters to `release_win32()` function makes it more difficult to maintain. But there is room for improvement (see other review comments).